lib/src/view_models/bluetooth_provisioning_flow_view_model.dart

```diff
@@ -110,15 +110,17 @@ class BluetoothProvisioningFlowViewModel extends ChangeNotifier {
     }
   }
 
-  Future<void> unlockBluetoothPairing(BuildContext context) async {
+  Future<bool> unlockBluetoothPairing(BuildContext context) async {
     try {
       isLoading = true;
-      await device?.unlockPairing();
+      await device?.unlockPairing(psk: _psk);
```

Comment:
also a fix here, have to pass in the provided `_psk` or it defaults to `viamsetup` in the package and fails, now that we're testing without using `viamsetup`